Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -265,6 +291,13 @@ def main():
         pr.add_to_assignees(security_reviewer)
         pr.add_to_labels(SECURITY_LABEL)
 
+    # adding TIM reviewer
+    packs_in_pr = packs_to_check_in_pr(pr_files)
+    if is_tim_reviewer_needed(packs_in_pr):
+        if support_label in (XSOAR_SUPPORT_LEVEL_LABEL, PARTNER_SUPPORT_LEVEL_LABEL):
```

Comment:
Create a new function called `is_tim_reviewer_needed` that will call `is_tim_content ` and will also check the conditions of the labels and categories and will return true or false